src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== Upgrade/Downgrade ProActive using Docker Compose
+
+The ProActive version could be easily upgraded using Docker Compose. To do so, we support automatic upgrade, downgrade, and backup of ProActive Server installations in Docker.
+Docker Compose configuration relies on Docker container volumes to host ProActive versions. The `proactive-default` volume hosts the current and default installation. The `proactive-previous` volume hosts the previous installation after upgrade.
+Finally, the `proactive-backup` volume hosts all previous installtion after multiple upgrades.
+In the following, we are going to explain each procedure.
+
+===== ProActive Upgrade
+In order to upgrade the ProActive Server to a newer version, you have to edit the `.env` config file to provide a newer version. The current ProActive version is specified using the `PROACTIVE_VERSION` option.
+For example, you can replace `PROACTIVE_VERSION=12.0.0` by `PROACTIVE_VERSION=12.0.8` to get a maintenance release of ProActive.
```

Comment:
Does this upgrade cover major and minor releases ? If yes could you add a mention about that ?
What happen if someone provide an anterior version like 11.0.0 ?

Reply:
ok